ajax/libs/ant-design-pro/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ant-design-pro",
+  "filename": "ant-design-pro.min.js",
+  "version": "1.1.0",
+  "description": "An enterprise-class UI design language and React-based implementation",
+  "license": "MIT",
+  "author": {
```

Comment:
How did you get this info? Please provide ref if you have.
It would be better to check with the upstream if this is suitable if you fill out this by your own.
Other parts LGTM.

Reply:
Because he host most commits, I consider he would be the author.

Reply:
@finch185277 I think it might be better to check this by opening an issue there. This repo belongs to organization @ant-design